santoshp210-akamai/manager
# upcoming: [DI-21340] - Alert details and overview changes

## Description 📝
Includes changes to view the show details page along with the alerts overview box

## Changes  🔄

1. Add navigation changes for show details page.
2. Add a show detail component and it's UT's
3. Add a overview box component and it's UT's
4. Add mocks and enhacce the factories

## Target release date 🗓️
03-10-2024

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![Screenshot 2024-12-02 at 11 02 18 AM](https://github.com/user-attachments/assets/d4ecce46-5b6a-478c-b654-433a93774b6a)|![Screenshot 2024-12-02 at 11 02 26 AM](https://github.com/user-attachments/assets/e328d423-27a9-437f-be94-7dc7cb726f9f)|
|![Screenshot 2024-12-02 at 11 44 04 AM](https://github.com/user-attachments/assets/0f3eeec9-883c-4f73-8d0f-06284fb4d9fb)|![image](https://github.com/user-attachments/assets/d2bb8ebf-4676-4ac4-8dd4-6b3f65f9fa9f)|

## How to test 🧪

1. Login as legacy mock, since api's are not available in the production
2. Go to monitor tab
3. Go to alerts sub tab
4. You will see list of alerts, click on action menu and click on show details option
5. You will be able to see show details page with overview box

## As an Author, to speed up the review process, I considered 🤔

👀 Doing a self review
❔ Our [contribution guidelines](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md)
🤏 Splitting feature into small PRs
➕ Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
🧪 Providing/improving test coverage
 🔐 Removing all sensitive information from the code and PR description
🚩 Using a feature flag to protect the release
👣 Providing comprehensive reproduction steps
📑 Providing or updating our documentation
🕛 Scheduling a pair reviewing session
📱 Providing mobile support
♿  Providing accessibility support

<br/>


## As an Author, before moving this PR from Draft to Open, I confirmed ✅


</details>

---
